docs/changelog.rst

```diff
@@ -9,6 +9,7 @@ unreleased
 * Update qiskit-ibm-runtime version to 0.13.0.
 * Update qiskit-aer version to 0.13.0.
 * Introduce dependency on qiskit-algorithms.
+* Add option for automatic incrementing seed on ``process_circuits()``.
```

Comment:
Update description in changelog.

Reply:
Done in [85489ef](https://github.com/CQCL/pytket-qiskit/pull/197/commits/85489ef983300e4739976ba68048e55e5ceb932a)